modules/applications/examples/ui-builder/development/docker-compose.yml

```diff
@@ -1,10 +1,10 @@
 # Use for running a bonita-ui-builder during development.
 services:
   bonita-ui-builder:
-    image: ${IMAGE_NAME:-pass:a[{bonita-ui-builder-image}]:latest}
+    image: pass:a[{bonita-ui-builder-image}]:latest
```

Comment:
**nitpick:** The `latest` tag doesn't exist.

Reply:
After analysing of the UIB promote job, we think it just a missing to promote the Docker image as `latest`.  
So, no need to change the documentation 😊